src/wrappers/python/PyOpenEXR.cpp

```diff
@@ -1974,7 +2270,11 @@ PYBIND11_MODULE(OpenEXR, m)
              py::arg("channels"),
              py::arg("name")="")
         .def("__repr__", [](const PyPart& p) { return repr(p); })
+#if XXX
```

Comment:
are these XXX things in this area not working yet?

Reply:
I meant to remove those entirely. I thought operator== would be helpful for testing, but it's way too hard in C++, much easier in python in the actual tests.